tests/scene/scene.tests.ts

```diff
@@ -502,27 +523,17 @@ describe('Scene', () =>
         //
         container.addChild(child2);
 
-        expect(container.renderGroup.childrenToUpdate[1]).toEqual({
-            list: [child, container2, child2],
-            index: 3
-        });
+        compareUpdateList([child], container.renderGroup, 1);
 
-        expect(container2.renderGroup.childrenToUpdate[1]).toEqual({
-            list: [child3],
-            index: 1
-        });
+        compareUpdateList([child3], container2.renderGroup, 1);
 
         child2.x = 110;
 
-        expect(container.renderGroup.childrenToUpdate[1]).toEqual({
-            list: [child, container2, child2],
-            index: 3
-        });
+        compareUpdateList([child], container.renderGroup, 1);
 
-        expect(container2.renderGroup.childrenToUpdate[1]).toEqual({
-            list: [child3],
-            index: 1
-        });
+        compareUpdateList([child3], container2.renderGroup, 1);
+
+        expect(true).toBeTrue();
```

Comment:
this seems wrong

Reply:
i know this looks a bit weird - its because the the lint rule specifies that there must be an expect function in the test. The expect is contained in the compareUpdateList function. I will disable the lint rule instead as that is a bit more reasonable!